ci/cscs-ci.yml

```diff
@@ -118,3 +163,61 @@ test py310:
     - SUBPACKAGE: next
       VARIANT: [-nomesh, -atlas]
       SUBVARIANT: [-cuda11x, -cpu]
+.test_helper_aarch64:
+  extends: [.container-runner-todi-gh200, .test_helper]
+  parallel:
+    matrix:
+    - SUBPACKAGE: [cartesian, storage]
+      VARIANT: [-internal, -dace]
+      SUBVARIANT: [-cuda12x, -cpu]
+    - SUBPACKAGE: eve
+    - SUBPACKAGE: next
+      VARIANT: [-nomesh, -atlas]
+      SUBVARIANT: [-cuda12x, -cpu]
+  before_script:
+  # TODO: remove start of CUDA MPS daemon once CI-CD can handle CRAY_CUDA_MPS
+  - CUDA_MPS_PIPE_DIRECTORY="/tmp/nvidia-mps" nvidia-cuda-mps-control -d
+  variables:
+    # Grace-Hopper gpu architecture is not enabled by default in CUDA build
+    CUDAARCHS: "90"
+    # limit test parallelism to avoid "OSError: too many open files"
+    NUM_PROCESSES: 32
```

Comment:
Do you know where the problem is coming from? Is this something we should report as feedback to the system?

Reply:
The problem was coming from the cmake build stage included in each test case. If we run pytest using the `auto` parallelism, that will run 288 tests in parallel, each test triggering a GT4Py build. I didn't look at this issue in depth, I just thought that we could limit this parallelism.

Reply:
I can remove this limitation and see it the same issue still appears.

Reply:
No, I think makes sense.